composer.json

```diff
@@ -29,7 +29,7 @@
         "phpunit/phpunit": "^9.3"
     },
     "conflict": {
-        "laravel/framework": "<9.19.0"
+        "laravel/framework": "<9.37.0"
```

Comment:
Taking a guess that this is the version we release this in. Will need to confirm on release day.